python/hsfs/feature_group.py

```diff
@@ -512,6 +512,22 @@ def get_parent_feature_groups(self) -> explicit_provenance.Links:
             `hsfs.client.exceptions.RestAPIError`.
         """
         return self._feature_group_engine.get_parent_feature_groups(self)
+    
+    def get_storage_connector(self):
```

Comment:
ok, had a talk with Jim on this, and he wants things as slim as possible and usable for the user.
I had to do the same for the model provenance.
Can we rename this method to `get_storage_connector_provenance` - this returns a Links object, a object that typical users will not be used to, only powerusers.
Can we create another wrapper method `get_storage_connector` where you extract the storage connector from `accessible`. Similar to: https://github.com/logicalclocks/feature-store-api/blob/master/python/hsfs/feature_view.py

For upstream links of this type, we know a fg can have at most one storage connector.
Maybe you can also check if accessible is empty, if there is something in delete or innaccesible and log a warning if there exists and just return None. I am planning to do similar for models in a future PR.